source/css/sass/mixins/_typography.scss

```diff
@@ -67,6 +67,21 @@
   vertical-align: middle;
 }
 
+@mixin _font-variant-position($position) {
+
+  @if $position == super {
+    top: -0.5em;
+  }
```

Comment:
Should error on `@else`, like https://github.com/libero/pattern-library/blob/01b6f55cd4fbdb775a6462834c3932695ac17288/source/css/sass/mixins/_spacing.scss#L172-L174